src/pybind/mgr/dashboard/module.py

```diff
@@ -15,6 +15,7 @@
 from uuid import uuid4
 from OpenSSL import crypto, SSL
 from mgr_module import MgrModule, MgrStandbyModule, Option
+import mgr_util
```

Comment:
i'd suggest
```py
from mgr_util import get_default_addr
```

to be more specific what function we are interested in when importing a module.

Reply:
ok